sdk_container/src/third_party/coreos-overlay/coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild

```diff
@@ -28,7 +28,7 @@ DEPEND="
 	app-text/mandoc
 	coreos-base/hard-host-depends
 	coreos-base/coreos-sb-keys
-	dev-libs/gobject-introspection
+	dev-libs/gobject-introspection-common
```

Comment:
I don't think this needs to be explicitly included here.

Reply:
I did a grep for dev-libs/gobject-introspection-common and it seems to be pulled in by three packages:

- dev-libs/glib - conditionally RDEPEND on USE=introspection, which we disable
- sys-auth/polkit - BDEPEND, this package seems to be built only for board
- sys-fs/udisks - BDEPEND, but we actually are not even building this package, so it should be dropped (not in this PR, though)

Seems like nothing would pull dev-libs/gobject-introspection-common through coreos-devel/sdk-depends if we removed the line from the ebuild. This would result in the package being built during board packages build instead of SDK build. Since (ideally) we want the SDK to provide all BDEPEND packages of board packages, this should stay.